spec/factories/gi_bill_status_response.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Neato 👍  this is good to have.

J/w - are the VCR cassettes up to date with this format? / Has it changed since those cassettes were recorded?

Reply:
I have not used VCR cassettes before